content/docs/command-reference/remove.md

```diff
@@ -1,6 +1,7 @@
 # remove
 
-Remove stage, `.gitignore` entry, and unprotect outputs.
+Remove stages or `.dvc` files, unprotect or delete their <abbr>outputs</abbr>,
+and erase the `.gitignore` entries.
```

Comment:
This should have a matching core repo update, probably.

Reply:
It's like a complete algorithm here, but what does this command do from the user perpective?

Reply:
we mention unprotect here but not in the Description - also doesn't sound right

Reply:
> hat does this command do from the user perpective?

The opposite of `add` or `run/repro` per stage output... ⌛ 

> we mention unprotect

True, I guess this doesn't unprotect outputs, it unlinks them rather (no matter which link type). Updating...

Reply:
Changed it to just "Stop tracking files or directories, and optionally delete them."

BTW more and more https://dvc.org/doc/user-guide/how-to/stop-tracking-data seems redundant... Should I just remove that how-to and include `dvc gc` in an example here in the remove ref?

Reply:
> Sent core update to iterative/dvc/pull/4999

Reply:
More updates here... Actually, I guess it's more than stopping tracking data in the case of stages: it removes the entire stage... So this command basically does different things depending on the targets you send it: for .dvc files or outputs it stops tracking the data; for stages it removes them form the pipeline.

Updated again, to "Remove stages from `dvc.yaml` and/or stop tracking files or directories (and optionally delete them)."

Reply:
> Should I just remove that how-to and include dvc gc in an example here in the remove ref?

don't have a strong opinion, but yes, they overlap a lot

how to might be useful since it's easier to find to my mind and a lot of people asking about it